examples/fluids/src/mat-ceed.c

```diff
@@ -1006,21 +1006,27 @@ PetscErrorCode MatCeedContextCreate(DM dm_x, DM dm_y, Vec X_loc, Vec Y_loc_trans
     PetscCall(DMRestoreLocalVector(dm_x, &dm_X_loc));
     if (X_loc) PetscCall(VecGetLocalSize(X_loc, &X_loc_len));
     PetscCallCeed((*ctx)->ceed, CeedVectorGetLength((*ctx)->x_loc, &ctx_x_loc_len));
-    if (X_loc) PetscCheck(X_loc_len == dm_x_loc_len, PETSC_COMM_SELF, PETSC_ERR_LIB, "X_loc must match dm_x dimensions");
-    PetscCheck(x_loc_len == dm_x_loc_len, PETSC_COMM_SELF, PETSC_ERR_LIB, "op must match dm_x dimensions");
-    PetscCheck(x_loc_len == ctx_x_loc_len, PETSC_COMM_SELF, PETSC_ERR_LIB, "x_loc must match op dimensions");
+    if (X_loc)
+      PetscCheck(X_loc_len == dm_x_loc_len, PETSC_COMM_SELF, PETSC_ERR_LIB,
+                 "X_loc (%" PetscInt_FMT ") must match dm_x (%" PetscInt_FMT ") dimensions", X_loc_len, dm_x_loc_len);
```

Comment:
Multi line if statement should be guarded with {}

Reply:
Ope, didn't pay attention to the clangformat changes.

Reply:
Ye, I was guessing that clang got ya there